Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerRoleStore.cs

```diff
@@ -24,11 +24,13 @@ public SqlServerRoleStore(IAuthorizationDbContext authorizationDbContext)
 
         public async Task<Role> Add(Role model)
         {
+            //model.Id = Guid.NewGuid();
```

Comment:
Do we need this commented out?

Reply:
I'm not sure. I would think we need it, but are Role tests passing without it?